### PR TITLE
Vagrant-based tests for infrastructure

### PR DESCRIPTION
**Pull request by joshmoore**
Used this infrastructure to test https://github.com/openmicroscopy/infrastructure/pull/57

First run showed:

screen shot 2016-03-14 at 10 59 02

Second run showed:

screen shot 2016-03-14 at 10 58 39

### CODE REVIEW

File: ansible/test.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+# Playbook for testing individual and sets of roles.
+#
+# Host names in this file should match the name of the Vagrant configuration
+# in Vagrantfile. Tests can be run either by using `vagrant up $NAME` when a
+# box does not yet exist, or `vagrant provision $NAME` for one that exists &
+# is running.
+
+- hosts: docker
+  sudo: True
```
Comment (manics) [inline ansible/test.yml]: Can you remove this? All tasks rewriting `sudo` should include `become: yes`

Comment (joshmoore) [inline ansible/test.yml]: Will do. I tried for a larger docker configuration which *did* require `sudo: True` so likely we're missing some `become: yes` lines in one of: docker-storage, docker-advanced, docker-dns-*

**Comment by joshmoore** [comment]
Force pushed.

**Comment by joshmoore** [comment]
Push forced again now that the samba role is merged.

**Comment by joshmoore** [comment]
Merging for integration with #61

**Pull request merged**